bees/ircbee/ircbeefactory.go

```diff
@@ -131,6 +131,57 @@ func (factory *IrcBeeFactory) Events() []bees.EventDescriptor {
 				},
 			},
 		},
+		{
+				Namespace:   factory.Name(),
+				Name:        "join",
+				Description: "A user joined a channel",
+				Options: []bees.PlaceholderDescriptor{
+					{
+						Name:        "user",
+						Description: "The username to watch",
+						Type:        "string",
+					},
+					{
+						Name:        "channel",
+						Description: "The channel the user should be watched",
+						Type:        "string",
+					},
+				},
+			},
+			{
+					Namespace:   factory.Name(),
```

Comment:
The indentation here got messed up. Always run ```go fmt *.go``` on your code before committing, to avoid such issues.